docs/permissions.md

```diff
@@ -0,0 +1,151 @@
+# Permissions
+
+`marsha` user access and permissions are managed following two different approaches:
+
+- For the LTI part, the authentication and authorization are managed by the LTI consumer site.
+  The LTI consumer site is responsible for authenticating the user and determining
+  their `role`. The user ID and Role are then passed to `marsha` through the LTI
+  request.
+- For the standalone site, the authentication and authorization are managed by `marsha`.
+  `marsha` can also rely on external SSO (see [renater_federation_saml.md](renater_federation_saml.md)).
+
+
+## LTI
+
+### Authentication
+
+The user is authenticated by the LTI consumer site, and `marsha` relies on the data provided by the signed LTI request.
```

Comment:
```suggestion
The user is authenticated by the LMS, and `marsha` relies on the data provided by the signed LTI request.
```

Reply:
`LTI consumer site` is accurate IMO. In richie we are consuming marsha in LTI and it is not a LMS. But you can argue that it's a edge case.